src/test/java/it/gov/pagopa/reporting/service/FlowServiceIntegrationTest.java

```diff
@@ -0,0 +1,161 @@
+package it.gov.pagopa.reporting.service;
+
+import com.azure.core.util.BinaryData;
+import com.azure.storage.blob.BlobClient;
+import com.azure.storage.blob.BlobContainerClient;
+import com.azure.storage.blob.BlobServiceClient;
+import com.azure.storage.blob.BlobServiceClientBuilder;
+import com.microsoft.azure.storage.CloudStorageAccount;
+import com.microsoft.azure.storage.RetryNoRetry;
+import com.microsoft.azure.storage.StorageException;
+import com.microsoft.azure.storage.table.CloudTable;
+import com.microsoft.azure.storage.table.CloudTableClient;
+import com.microsoft.azure.storage.table.TableRequestOptions;
+import it.gov.pagopa.reporting.model.Flow;
+import lombok.SneakyThrows;
+import org.junit.ClassRule;
+import org.junit.jupiter.api.Test;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+import org.testcontainers.utility.DockerImageName;
+
+import java.io.InputStream;
+import java.util.List;
+import java.util.logging.Logger;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.mockito.Mockito.spy;
+
+@Testcontainers
+class FlowServiceIntegrationTest {
+    @ClassRule
+    @Container
+    public static GenericContainer<?> azurite = new GenericContainer<>(
+            DockerImageName.parse("mcr.microsoft.com/azure-storage/azurite:latest")).withExposedPorts(10001, 10002,
+            10000);
+
+    Logger logger = Logger.getLogger("testlogging");
+
+    String storageConnectionString = String.format(
+            "DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://%s:%s/devstoreaccount1;TableEndpoint=http://%s:%s/devstoreaccount1",
```

Comment:
Alert dismissed because `storageConnectionString` is the emulator connection string value, used only for local test.
For more info see: https://learn.microsoft.com/en-us/azure/storage/common/storage-use-emulator